hack/verify-glog-format-strings.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+# Copyright 2017 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+KUBE_ROOT=$(dirname "${BASH_SOURCE}")/..
+
+cd "${KUBE_ROOT}"
+result=0
+
+checkGlog() {
+  while read file; do
+    if grep -q "github.com/golang/glog" $file; then
+      if grep "glog.Warning(\"" $file | grep %; then
+        echo $file
+	let result+=1
+      fi
+      if grep "glog.Info(\"" $file | grep %; then
+        echo $file
+	let result+=1
+      fi
+      if grep "glog.Error(\"" $file | grep %; then
+        echo $file
+	let result+=1
+      fi
+      if grep ").Info(\"" $file | grep % | grep glog.V\(; then
+        echo $file
+	let result+=1
+      fi
+    fi
+  done
+  return ${result}
```

Comment:
We need to make the output message more friendly.
You need to tell users: 

    * What's wrong
    * How to fix it

Reply:
Copy that